pandas/tseries/timedeltas.py

```diff
@@ -32,6 +32,25 @@ def to_timedelta(arg, unit='ns', box=True, errors='raise', coerce=None):
     Returns
     -------
     ret : timedelta64/arrays of timedelta64 if parsing succeeded
+
+    Examples
+    --------
+
+    Parsing a single string to a Timedelta:
+    >>> pd.to_timedelta('1 days 06:05:01.00003')
```

Comment:
Small formatting note: there needs to be a blank line between the two lines above (between the code blocks and the text, so the code blocks are rendered as code, see https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#sections for a full reference)